src/borg/remote.py

```diff
@@ -711,6 +713,19 @@ def call_many(self, cmd, calls, wait=True, is_preloaded=False, async_wait=True):
         if not calls and cmd != 'async_responses':
             return
 
+        def fill_send_buffer():
```

Comment:
why is this called **fill**_send_buffer? looks like it is rather sending (and emptying) the send buffer.

Reply:
It is filling pipe send buffer from internal send buffer... But you are right, from here it looks like we are emptying buffer, I'll just rename it `send_data`?

Reply:
yeah, send_data please.